src/main/resources/org/biouno/unochoice/stapler/unochoice/UnoChoice.es6

```diff
@@ -363,7 +363,7 @@ var UnoChoice = UnoChoice || ($ => {
                 $(".behavior-loading").show();
                 // start updating in separate async function so browser will be able to repaint and show 'loading' animation , see JENKINS-34487
                 setTimeout(() => {
-                   _self.cascadeParameter.update();
+                   _self.cascadeParameter.update(false);
```

Comment:
The other changes were easier to review. This one is the only one I didn't understand straight away. Could you explain why the `false` added here?

Reply:
Sorry about that. The default value was `undefined` which was interpreted as `false`. This makes it explicit.